DQMOffline/Muon/src/MuonTiming.cc

```diff
@@ -0,0 +1,196 @@
+#include "DQMOffline/Muon/interface/MuonTiming.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h" 
+#include "DataFormats/MuonReco/interface/MuonEnergy.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include <string>
+#include "TMath.h"
+using namespace std;
+using namespace edm;
+
+
+
+MuonTiming::MuonTiming(const edm::ParameterSet& pSet) {
+  parameters = pSet;
```

Comment:
it looks like parameters is used only in the constructor. If so, no need to keep it in memory for the full process